src/scales/scale.time.js

```diff
@@ -36,10 +40,94 @@ module.exports = function(Chart) {
 			},
 		},
 		ticks: {
-			autoSkip: false
+			autoSkip: false,
+			mode: 'series',   // 'linear|series'
```

Comment:
I think `series` and `auto` is perhaps not the best default because I think that most of the time using `series` it'd probably be used with `linear`. I think `auto` will probably be the better option most of the time when `linear` is used. So I'd probably change on or the other of the defaults here.

`linear` is the current default on the timescale. I'm assuming `linear` & `auto` is supported eventhough there's no screenshot of that one. That'd be my vote for combination of options for the default